README.rst

```diff
@@ -0,0 +1,15 @@
+===========
+Python Prep
+===========
+
+The program ``pythonprep_assignment.py`` is an exercise for students.
+It is a Python exercise designed to test some basic Python skills.
+Our training course includes Python programming and the student needs to be
+able to program in it.
+
+Students are given the file, and need to read the file into a string, and 
+then break the string into numbers, and then sum the numbers.
+
+To run your program type python pythonprep_assignment.py. You have successfully 
```

Comment:
inline code. ` ``python pythonprep_assignment.py`` `

Reply:
Yes, thank you.